### PR TITLE
Single tenant update

### PR DESCRIPTION
**Pull request by code4clouds**

### CODE REVIEW

**Comment by santhoshb-msft** [comment]
image
we might need to update the setting @code4clouds

**Comment by code4clouds** [comment]
> image
> we might need to update the setting @code4clouds , mtclientid, mtclientsecret (do we need this?)

Yes...  I added that as it is the multi-tenant AAD for the SSO.

**Comment by santhoshb-msft** [comment]
> > image
> > we might need to update the setting @code4clouds , mtclientid, mtclientsecret (do we need this?)
> 
> Yes... I added that as it is the multi-tenant AAD for the SSO.

Sorry two things 
1. we need to update the settings, remove _
2. Do we need clientsecret for multi-tenant auth?

File: src/SaaS.SDK.CustomerProvisioning/appsettings.json [hunk 1]
```diff
@@ -11,8 +11,10 @@
 
   //  "granttype": "client_credentials",
   //  "clientid": "",
-  //  "tenantid": "",
   //  "clientsecret": "",
+  //  "mt_clientid": "",
```
Comment (santhoshb-msft) [inline src/SaaS.SDK.CustomerProvisioning/appsettings.json:15]: just referring to this typo

**Pull request closed without merging**